bedrock/security/models.py

```diff
@@ -67,7 +67,7 @@ def save(self, force_insert=False, force_update=False,
 class SecurityAdvisory(models.Model):
     id = models.CharField(max_length=8, primary_key=True, db_index=True)
     title = models.CharField(max_length=200)
-    impact = models.CharField(max_length=100)
+    impact = models.CharField(max_length=100, null=True)
```

Comment:
Seems like there's not a big difference, but I think blank=true makes the most sense. I had to adjust some other things (avoid explicitly setting the field value to `None` e.g.), but I think this is the Right Way™️. Good catch.